libazureinit/src/media.rs

```diff
@@ -79,7 +84,23 @@ pub const PATH_MOUNT_POINT: &str = "/run/azure-init/media/";
 const CDROM_VALID_FS: &[&str] = &["iso9660", "udf"];
 const MTAB_PATH: &str = "/etc/mtab";
 
-// Get a mounted device with any filesystem for CDROM
+/// Retrieves a list of mounted devices with CDROM-type filesystems.
+///
+/// # Arguments
+///
+/// * `path` - Optional path to the mount table file.
+///
+/// # Returns
+///
+/// A `Result` containing a vector of device paths as strings, or an `Error`.
+///
+/// # Example
+///
+/// ```
+/// use libazureinit::media::get_mount_device;
+///
+/// let devices = get_mount_device(None).unwrap();
+/// ```
```

Comment:
You can avoid unwrap like below:
```
/// ```
/// fn main() -> Result<(), Box<dyn std::error::Error>> {
///   use libazureinit::media::get_mount_device;
///
///   let devices = get_mount_device(None)?;
///   Ok(())
/// }
/// ```

```
Please see the [section](https://rust-lang.github.io/api-guidelines/documentation.html#c-question-mark) of Rust API guide to know why `?` is better than `unwrap`.